### DISCUSSION TITLE
Feature Selection NOT prompt

### DISCUSSION THREAD

**Discussion by Alessandro1979** [serenity-is/Serenity#6263] [Help]
Hi,
I've a strange problem: when I start a new project, Serenity not longer prompt Feature Selection, but create directly the projet.
So I have, for example, northwind db included even if i don't need it.
Can you help me ?

Visual Studio 16.11.10
.Net Frmwork 4.8.04084
Serene 5.0.5.1.8

**Comment by wezmag** [comment]
Feature Selection was removed, but I think they will add it back soon.
In the mean time, you can remove the samples using NuGet Package Manager.

**Reply by Alessandro1979** [reply]
Thank you for the answer. 
But the problem is the code: It's generated with northwind manage embedded in code :/

**Reply by VictorTomaili** [reply]
The project has some NuGet packages which include `demo` in their names. And you need to remove some code blocks from `Dashboard` files and `startup.cs`. There is no problem with it. The feature selection tool does the same thing to eliminate some manual steps when starting a new project.

**Reply by wezmag** [reply]
@VictorTomaili Do you plan to add the feature selection tool back to the SERENE template?

**Reply by Alessandro1979** [reply]
@VictorTomaili thank you, It's work, just a clarification: also in DataMigrations.cs it's necessary remove/comment some code blocks.
